pkg/models/task_search.go

```diff
@@ -34,6 +34,61 @@ import (
 	"xorm.io/xorm/schemas"
 )
 
+type SubTableFilter struct {
+	Table           string
+	BaseFilter      string
+	FilterableField string
+	AllowNullCheck  bool
+}
+
+type SubTableFilters map[string]SubTableFilter
+
+var subTableFilters SubTableFilters = SubTableFilters{
+	"labels": {
+		Table:           "label_tasks",
+		BaseFilter:      "tasks.id = task_id",
+		FilterableField: "label_id",
+		AllowNullCheck:  true,
+	},
+	"label_id": {
+		Table:           "label_tasks",
+		BaseFilter:      "tasks.id = task_id",
+		FilterableField: "label_id",
+		AllowNullCheck:  true,
+	},
+	"reminders": {
+		Table:           "task_reminders",
+		BaseFilter:      "tasks.id = task_id",
+		FilterableField: "reminder",
+		AllowNullCheck:  true,
+	},
+	"assignees": {
+		Table:           "task_assignees",
+		BaseFilter:      "tasks.id = task_id",
```

Comment:
If this is always the same, would it make sense to add this as fallback?

On the other hand, it might be good to have it explicit.